dev-setup/cleanup.sh

```diff
@@ -26,4 +26,6 @@ kubectl delete namespace local-spark-operator | echo "local-spark-operator names
 
 minikube ssh "sudo rm -rf /tmp/hostpath-provisioner/local-mariadb /tmp/hostpath-provisioner/local-ozone"
 
+rm bpg-config.yaml
```

Comment:
Remove the generated config along with all the local stack since it will be no longer useful.